### PR TITLE
Move cc icons to the right code point

### LINKED ISSUE #92: Move CC license icons to the right code points
## Description
Since CC licenses have been included in the Unicode Standard, we can expect more and more fonts to start supporting the license icons in the future. Having the icons at the right code-points would mean that they can then be easily copied and pasted into a different document using a supported font while working as expected.

## Additional context
The [blog post by Eric Steuer](https://creativecommons.org/2020/03/18/the-unicode-standard-now-includes-cc-license-symbols/) that prompted this issue contains more information and the code-points for the license icons.

## Implementation

- [ ] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Tanuj22**
## Fixes

Fixes #92 by @dhruvkb 

## Description
Move cc icons to the right code point
## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Tanuj22** [comment]
Screenshot from 2020-06-12 23-18-33
The CC part isn't true now. Should I change or remove it?

**Comment by dhruvkb** [comment]
@Tanuj22 update it.

**Comment by Tanuj22** [comment]
@dhruvkb Update it to `\ec** & \1f1**`?

**Review by dhruvkb — CHANGES_REQUESTED** [review]
[You missed CC-ND](https://www.ctrl.blog/entry/creative-commons-unicode-fallback-font.html). Once you add it, you can classify them into standard and non-standard categories in the code-points table.

**Comment by Tanuj22** [comment]
Screenshot from 2020-06-13 03-42-01
@dhruvkb Please check this out. Although `CC-ND (229c)` doesn't fit in any of them

File: src/styles/vocabulary_icons.scss [hunk 1]
```diff
@@ -61,19 +61,19 @@ $icons: (
   "plus": "\ea0d",
   "minus": "\ea0e",
 
-  "cc-logo": "\ec00",
```
Comment (dhruvkb) [inline src/styles/vocabulary_icons.scss:64]: Please also arrange them in order of code point, separating the standard from the non-standard with a blank line.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Almost ready!

**Comment by dhruvkb** [comment]
Just put CC-ND with CC Standard and change the table column to `1f1**` + `229c`. Actually the already existing [Circled Equals](https://www.fileformat.info/info/unicode/char/229c/index.htm) sign was so similar to CC-ND, I think they just repurposed it.

**Review by dhruvkb — APPROVED** [review]

File: src/stories/vocabulary_icons.stories.mdx [hunk 2]
```diff
@@ -117,80 +117,66 @@ smallest possible file size bundle.
 
 ## CC licenses
 
-### Aspects
+### Standard
+
+The CC standard icons include those that have been added to the Unicode Standard in version 13.0. Do note the the CC-ND
```
Comment (Dishebh) [inline src/stories/vocabulary_icons.stories.mdx:122]: `Do note the the` -> `Do note that the`, I guess :sweat_smile:

**Pull request merged**